fe-vite-react/src/SingleArticlePage.jsx

```diff
@@ -0,0 +1,63 @@
+import React, { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import { getSingleArticle, getArticleComments, formatCommentDate } from "./utils";
+
+const SingleArticlePage = () => {
+  const { article_id } = useParams();
+  console.log("lskdfj", article_id);
+  const [comments, setComments] = useState([]);
+  const [singleArticle, setSingleArticle] = useState({});
+
+  useEffect(() => {
+    getSingleArticle(article_id)
+      .then((article) => {
+        setSingleArticle(article);
+      })
+      .catch((error) => {
+        console.log("error", error);
+      });
+    getArticleComments(article_id)
+      .then((comments) => {
```

Comment:
Try not to leave console.logs lingering. Once they have served their purpose, delete them.